bcgov/mds
# Allow permition deletion if the permit is not generated in Core.

# Main

-Allow users in Core to delete original and permit amendments if they are not generated in Core.  Before the user was getting this message informing that he/she was not able to delete the permit:
![image](https://user-images.githubusercontent.com/10526131/150233484-2f61c36e-cdbb-459f-8e21-cc36c0604ba7.png)


# Other


# How to test
- Create a permit amendment related to a NoW:
  a) Go to Mines and filter by Status Pending Verification. Click on Open
  b) Change the name for the mine you want to use o leave the existing one
  c) Solve all the issues in Process tab and issue a permit amendment

- Confirm you have the permit amendment under Permit tab which is assign to the original permit:
![image](https://user-images.githubusercontent.com/10526131/150233351-4b6eefcd-4365-4819-b312-e537403a2a97.png)
  a) Update is_generated_in_core to false in permit_amendment table for the record associated with the mine and permit_id 

- Delete the permit amendment:
![image](https://user-images.githubusercontent.com/10526131/150234262-b9415883-f120-47f4-a377-cd14e9f5990a.png)

- Confirm soft delete
  a) Confirm in permit_amendment table the field deleted_ind = True
  
![image](https://user-images.githubusercontent.com/10526131/150234533-1e4460ce-81d2-46e5-ac58-90268d4b151d.png)

# Notes
https://bcmines.atlassian.net/browse/MDS-3669
